### ISSUE TITLE
Triage radio button issue experienced on JAWS

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#306]
Research participant was unable to progress to the next screen when interacting with radio buttons on the [Public Service Award of Excellence](https://forms-staging.cdssandbox.xyz/id/1) demo form.

**Comment by srtalbot** [comment]
User profile: Government-user providing feedback on accessibility considerations.

Form: https://forms-staging.cdssandbox.xyz/id/1

Description of issue:

- The participant was using JAWS to review the form, and had significant difficulty understanding what was on the page, as it was saying that radio button selections were invalid and not advancing to the next item of focus as expected.
- The grouping in the radio buttons were off; the participant was unable to hear how many items were in the list. 

Triaging: Moderate to critical, requires more investigation.

**Comment by brdunfield** [comment]
I did some investigating, and used VoiceOver to compare our radio buttons to some example radio button examples on a11y documentation sites. I can definitely spot the difference, in our app it doesn't read out the grouping at all compared to the examples. Hard to describe over text, but the point is I can isolate the problem pretty easily without needing any extra or outside help.

I will try to do an effort estimation shortly as I learn more about what could be causing this!
